src/components/action-popover/action-popover-menu-button/action-popover-menu-button.component.tsx

```diff
@@ -11,7 +11,8 @@ import { IconType } from "../../icon";
 
 export type ActionPopoverMenuButtonAria = {
   "aria-haspopup": string;
-  "aria-label": string;
```

Comment:
question: why are we removing aria-label here?

Should we not be keeping it, as that's what is passed into the button and as far as I can see it's still supported, at least in the prop table